openmc/stats/multivariate.py

```diff
@@ -0,0 +1,543 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
As of python 3.4 you can use `from abc import ABC` and create abstract classes like `class UnitSphere(ABC)`. Just a touch simpler.